
	Don't use first byte of string as a pointer.
	* src/qemu_conf.c (qemudReportError): Use the pointer, errorMessage,
	not its first byte, errorMessage[0].

Signed-off-by: Jim Meyering <meyering@redhat.com>
---
 src/qemu_conf.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/qemu_conf.c b/src/qemu_conf.c
index eead0bc..e54da5b 100644
--- a/src/qemu_conf.c
+++ b/src/qemu_conf.c
@@ -68,7 +68,7 @@ void qemudReportError(virConnectPtr conn,
         errorMessage[0] = '\0';
     }

-    virerr = __virErrorMsg(code, (errorMessage[0] ? errorMessage[0] : NULL));
+    virerr = __virErrorMsg(code, (errorMessage[0] ? errorMessage : NULL));
     __virRaiseError(conn, dom, net, VIR_FROM_QEMU, code, VIR_ERR_ERROR,
                     virerr, errorMessage, NULL, -1, -1, virerr, errorMessage);
 }
